schemas/ForcedSource.yaml

```diff
@@ -95,8 +95,9 @@ calexpFlags:
     - base_PixelFlags_flag_saturatedCenter
     - base_PixelFlags_flag_crCenter
     - base_PixelFlags_flag_suspectCenter
-    - base_PixelFlags_flag_streak
-    - base_PixelFlags_flag_streakCenter
+    # Streak flags not yet propagated from difference imaging.
```

Comment:
If these are being left in but commented, please add a `TODO` and a DM ticket number for their eventual reinstatement.